ducktape/tests/test.py

```diff
@@ -133,6 +134,9 @@ def _escape_pathname(s):
     return re.sub("^\.|\.$", "", s)
 
 
+CLUSTER_SIZE_KEYWORD = "num_nodes"
```

Comment:
Again, no strong preference. Just kinda weird that the docstring explaining where you would actually use this name (as a parameter to the decorator) and the string version of it are defined in separate modules.